libres/lib/enkf/active_list.cpp

```diff
@@ -150,41 +94,70 @@ active_mode_type active_list_get_mode(const active_list_type *active_list) {
    scope to not dereference the NULL pointer.
 */
 
-const int *active_list_get_active(const active_list_type *active_list) {
-    if (active_list->mode == PARTLY_ACTIVE)
-        return int_vector_get_const_ptr(active_list->index_list);
+const int *ActiveList::active_list_get_active() const {
+    if (this->m_mode == PARTLY_ACTIVE)
+        return this->m_index_list.data();
     else
-        return NULL;
+        return nullptr;
+}
+
+const std::vector<int> &ActiveList::index_list() const {
+    return this->m_index_list;
 }
 
-void active_list_summary_fprintf(const active_list_type *active_list,
```

Comment:
Dont think this is used (only in `local_ministep_summary_fprintf`, which is not used)